dev/ese/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+set(MSVC_ESE_LINK_OPTIONS
```

Comment:
Aside ... is it standard for CMakeLists to be .txt files?  Because it really sucks that this extension mixes in with primary data files.  It feels like it should have it's own extension, .cmk or something.